aiida_wannier90/__init__.py

```diff
@@ -16,9 +20,9 @@
 ## If upgraded, remember to change it also in setup.json (for pip)
 __version__ = "2.0.0a1"
 
-from . import io
-from . import parsers
-from . import orbitals
+#from . import io
```

Comment:
What's the reason for not importing the submodules here? Too much startup time?

In any case: If we remove it, I think the code should be deleted, not just commented out.

Reply:
I commented them just to avoid complains about "unused imports" and I'm not sure if we need/want to keep it.

Reply:
Ah, I see. The imports here are unused, but on purpose (generally, that's fine in ``__init__.py`` files). The difference this makes is that with the imports the following would work:

```python
import aiida_wannier90
aiida_wannier90.io.write_win(...)
```

whereas otherwise you would have to explicitly use
```python
import aiida_wannier90.io
```
or 
```python
from aiida_wannier90.io import write_win
```

Typically I tend to add these imports because it's a bit more convenient to use, but it has a cost in taking longer to initially load the module. That is important mostly if the package is imported somewhere in a command-line tool.

Here I'm fine either way, but it should be a conscious decision.

Tagging @giovannipizzi for comment.

Reply:
I discussed with Giovanni and he agrees on keeping the imports and just disable the warnings. Otherwise he suggested to use   ```__all__``` but he was not sure about how it works. What do you think?

Reply:
Ok, yeah I agree there should be an ``__all__`` here. That might even solve the pylint warnings, but I'm not sure about that.

It should be
```python
__all__ = ("io", "parsers", "orbitals")
```
after the import statements. This determines what gets imported on ``from aiida_wannier90 import *``, and some tools (e.g. ipython) also take it into account for autocompletion.

Looking around the code, it seems we're missing ``__all__`` in a few places.. I'll make an issue for myself.